### PR TITLE
Mention indices counting from 0.

### PR DESCRIPTION
**Pull request by Robaert-codes**
As that is not mentioned before in the fundamentals.

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/java/fundamentals/operators-ii/from-is-included-while-to-is-excluded.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/operators-ii/from-is-included-while-to-is-excluded.md?ref=31316d41dc2454c4e28b1582cc05470757f326e7)

File: java/fundamentals/operators-ii/from-is-included-while-to-is-excluded.md [hunk 1]
```diff
@@ -38,8 +38,10 @@ links:
 
 As a convention, when specifying a range of values, the first value 'from' is included while the last value 'to' is excluded. 
 
-For example in `String.substring(from, to)`, the character at position 'from' will be included in the substring, while the character 'to' will not. 
+For example in `String.substring(from, to)`, the character at position 'from' will be included in the substring, while the character 'to' will not.
+
+Indices count from 0, as common in many programming languages.
```
Comment (kapnobatai137) [inline java/fundamentals/operators-ii/from-is-included-while-to-is-excluded.md:43]: Thanks for the suggestion @Robaert-codes. How does this look? (the `>` formats the text in-app as being a quote)

 ```suggestion
> **Remember:** As with most programming languages, indexes in Java start from 0.
```

**Comment by kapnobatai137** [comment]
I'm also working on improving the first workouts for Java, so I'll make sure to mention indexing there.

**Comment by Robaert-codes** [comment]
Great job, thanks!

**Comment by kapnobatai137** [comment]
Changes are live 🎉🎉. Thanks again for the contribution @Robaert-codes!

**Pull request merged**